### DISCUSSION TITLE
Any way to shortcut the tabs view like bookmarks and history?

### DISCUSSION THREAD

**Discussion by Crozh** [piroor/treestyletab#3297] [General]
I'd like to use the keyboard to control my tabs view in the same way I can CTRL+H to view history and then make the extension disappear if I press again. Is there a way to do this? If it is possible I haven't found out yet.

**Comment by irvinm** [comment]
You mean "F1"?

![image](https://user-images.githubusercontent.com/979729/221875061-6229d58c-b6c3-4f3a-8013-d5cd73032b00.png)

**Reply by Crozh** [reply]
Damn I don't know how this flew over my head. I guess I expected F1 to send me to some help tab and never pressed it.  
Thanks.

**Reply by vonHabsi** [reply]
How do you bring up this page?

**Reply by irvinm** [reply]
![image](https://github.com/user-attachments/assets/cc859bf5-0572-44c0-903f-cc101a9d225e)
